greeting-guestbook/README.md

```diff
@@ -0,0 +1,38 @@
+# DBOS Guestbook Hello App
+
+This is the app described in the [DBOS Programming Guide](https://docs.dbos.dev/getting-started/quickstart-programming)
+
+## Getting Started
```

Comment:
The first command here is `npm install` to install all the dependencies.

Reply:
done